### DISCUSSION TITLE
Punishment system

### DISCUSSION THREAD

**Discussion by RAVAN666Devil** [jagrosh/Vortex#93] [Questions] [answered]
How can i remove the pre-existing punishments. I want to add everything according to my will. Can i do that ? Is there any command to to do this ? 

Answer me plz

**Comment by jagrosh** [comment, marked as answer]
Setting punishments is outlined on [this page on the wiki](https://github.com/jagrosh/Vortex/wiki/Strikes#-how-do-i-set-what-punishments-are-given-at-different-numbers-of-strikes).
